examples/pointcloud.html

```diff
@@ -182,6 +182,14 @@ <h3>Available pointclouds:</h3>
                     pointcloud.protocol = 'potreeconverter';
                     pointcloud.url = serverUrl;
                     pointcloud.table = lopocsTable;
+                    if (new URL(window.location.href).searchParams.get('material') === 'three') {
```

Comment:
nits: `new URL(...).searchParams` is used 2 times, let's define a `var searchParams` on top.

Reply:
done (9529b2c)